app/src/controllers/administrate/projects/sagas.js

```diff
@@ -116,7 +116,7 @@ function* watchAddProject() {
 
 function* deleteProject({ payload: project }) {
   try {
-    yield call(fetch, URLS.project(project.id), {
+    yield call(fetch, URLS.projectByName(project.id), {
```

Comment:
Looks like it was an unexpected usage of endpoint with `projectName`, because here we use a project `id`.
Let's use it as `URLS.project([project.id])`.
```suggestion
    yield call(fetch, URLS.project([project.id]), {
```

Reply:
Done.